static/app/views/explore/hooks/useChartBoxSelect.tsx

```diff
@@ -64,6 +74,14 @@ export function useChartBoxSelect({
     0
   );
 
+  const onBrushStart = useCallback<EChartBrushStartHandler>(() => {
+    // Echarts either lets us connect all interactivity of the charts in a group or none of them.
+    // We need connectivity for cursor syncing, but having that enabled while drawing, leads to a
+    // box drawn for all of the charts in the group. We are going for chart specific box selections,
+    // so we disconnect the group while drawing.
+    echarts?.disconnect(groupName);
```

Comment:
seems kind of awkward but I cant think of a better way to handle this in echarts right now

Reply:
Yeah, would require a change in the echarts repo, for doing something like this properly atm.